src/components/About/About.css

```
@@ -4,4 +4,57 @@
   align-content: center;
   align-items: center;
   min-height: 300px;
+  flex-direction:column;
 }
+
+.about_bg{
+  background-image: url('./images/ss_2.jpg');
+  width: 100%;
+  height: 400px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right;
```

```suggestion
  background-position: right;
  border-radius: 5px;
```

Minor edit to maintain a normalized styling throughout the website.